src/cljs/todo_app_two/core.cljs

```diff
@@ -80,4 +109,5 @@
 (defn init! []
   (start-router!)
   (ajax/load-interceptors!)
+  (rf/dispatch [:create-todos])
```

Comment:
There's a function with the name "create-todos".
It's not clear how that differs from this event